web_src/less/_repository.less

```diff
@@ -3008,6 +3008,112 @@ td.blob-excerpt {
   }
 }
 
+.image-diff-container {
+  text-align: center;
+  padding: 30px 0;
+
+  img {
+    border: 1px solid var(--color-primary-light-7);
+    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAG0lEQVQYlWN4+vTpf3SMDTAMBYXYBLFpHgoKAeiOf0SGE9kbAAAAAElFTkSuQmCC) right bottom var(--color-primary-light-7);
```

Comment:
I wonder if this is the best way for the background image ...
![aaa](https://user-images.githubusercontent.com/24977596/105870860-ff055600-5ff8-11eb-9144-d82c41b5a8e8.png)

Reply:
I thought it is small enough to use it inline instead of wasting a request.
What do you think about a variable like in
https://github.com/go-gitea/gitea/blob/b2c20b68a08e23bc952402a553d59a4613188bd0/web_src/less/_base.less#L114-L115
In this case it could be overwritten by themes. I could move the background image to a seperate class. Then it would be reusable for other pictures like the normal file view.

Reply:
No its fine